### ISSUE TITLE
Unable to get reports from SASReportDescriptor with name and location

### ISSUE THREAD

**Issue by Highbugs** [sassoftware/sas-sdk-for-iOS-examples#8]
I'm trying to use the SASReportDescriptor.init(name: String, location:String) but it doesn't work at all. Imagine that my report is in 'Folder 1/SubFolder/Report'. To get this report I initiate a sas report descriptor this way `SASReportDescriptor(name: "Report", location: "/Folder 1/SubFolder")` but it doesn't work in the same. On the console it returns `CONN_UNSUPPORTED_URL`.

**Comment by rchogan** [comment]
Hi Highbugs!   Im sorry this API is not working as expected. 

A couple of questions to help diagnose:

- What version of the SDK are you using?
- What version of VA are you connecting to?

**Comment by Highbugs** [comment]
- SDK Version : 8.14.1
Related with VA I don't know what is it, neither how to obtain its version.

**Comment by rchogan** [comment]
In your browser, can you go to https://servername.host.com/SASVisualAnalyticsTransport/ (where you replace the server with your servername)?   If you are on VA 7.x, then you will see the version # there.   If not, you are on 8.x, and you can try http://servername.host.com/SASReportViewer/apiMeta.  This will give you version information

**Comment by Highbugs** [comment]
I've got that. My VA is 8.2.

**Comment by rchogan** [comment]
Highbugs, 
There appears to be a bug with that API against 8.x VA servers.   I have reported a defect, and we will get that fixed.   Can you use the UUID API on SASReportDescriptor for now?

**Comment by Highbugs** [comment]
Yes. Currently I'm getting the reports by UUID and it works well.

**Comment by rchogan** [comment]
Highbugs, thanks for pointing this out.  We have reported this internally, and are working to address it.  I will update here when we have released a fix.
